src/main/java/frc/utils/allianceutils/AllianceUtils.java

```diff
@@ -57,10 +51,7 @@ private static Pose2d mirror(Pose2d pose) {
      * @return the rotation
      */
     public static Rotation2d toMirroredAllianceRotation(Rotation2d rotation) {
-        if (isBlueAlliance()) {
-            return rotation;
-        }
-        return new Rotation2d(Math.PI).minus(rotation);
+        return isBlueAlliance() ? rotation : new Rotation2d(Math.PI).minus(rotation);
```

Comment:
isn't there a constant for half a circle somwhere?

Reply:
Why, it I will not help here because in this calc 0 become 180 but 90 stay 90